### ISSUE TITLE
Reverse proxy

### ISSUE THREAD

**Issue by aolney** [aolney/mofacts-automated-authoring#12]
- [x] Service subdomains
- [x] Caddy as systemd with subdomain reverse proxy 
- [x] Dockers reconfigured to use subdomains

**Comment by aolney** [comment]
Caddy conflicts with traefik in TLJH installation, so replacing caddy with traefik

**Comment by aolney** [comment]
Easily implemented in `traefik` 1.7 that ships with TLJH. Just don't call `tljh-config reload proxy` or it will overwrite the `rules.toml` file below

```
[frontends.frontend__2F]
backend = "backend__2F"
passHostHeader = true

[frontends.allennlp]
backend = "allennlp"
passHostHeader = true

[frontends.spacy]
backend = "spacy"
passHostHeader = true

[frontends.frontend__2F.routes.test]
rule = "PathPrefix:/"
data = "{\"hub\": true}"

[frontends.allennlp.routes.rule1]
rule = "Host: allennlp.olney.ai"

[frontends.spacy.routes.rule1]
rule = "Host: spacy.olney.ai"

[backends.backend__2F.servers.server1]
url = "http://127.0.0.1:15001"
weight = 1

[backends.allennlp.servers.server1]
url = "http://127.0.0.1:8002"
weight = 3

[backends.spacy.servers.server1]
url = "http://127.0.0.1:8001"
weight = 2
```
